packages/libs/snap/snap.manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "version": "1.0.0",
+  "description": "Metamask snap to manage your Kadena accounts and KDA.",
+  "proposedName": "Kadena",
+  "source": {
+    "shasum": "3sN/k/rB5ppkYbr/mtopFy35kLWUro9Sl44z28LhHa8=",
+    "location": {
+      "npm": {
+        "filePath": "dist/bundle.js",
+        "iconPath": "images/icon.svg",
+        "packageName": "@kadena/snap",
+        "registry": "https://registry.npmjs.org/"
+      }
+    }
+  },
+  "initialPermissions": {
+    "snap_dialog": {},
+    "endowment:page-home": {},
+    "snap_getBip32Entropy": [
+      {
+        "path": [
+          "m",
+          "44'",
+          "626'"
+        ],
+        "curve": "ed25519"
+      }
+    ],
```

Comment:
Is this the hardened variant of the index? I don't see the `<index>` marked anywhere. Ie is it generating the same keys as `m/44'/626'/${index}'` from https://github.com/kadena-community/kadena.js/blob/main/packages/libs/hd-wallet/docs/decisions/0001-use-slip10-for-private-key-generation.md

Reply:
Edit: it looks like it's using the hardened index: packages/libs/snap/src/utils/kadenaUtils.ts